scripts/benchmark_conversion.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
```

Comment:
Is this included by accident? It seems like a nice thing to have checked-in, but only tangentially related to the rest of the PR..

Reply:
It was an accident, but then purposefully left in. Seemed useful to have around.

Reply:
👍 seems reasonable to me — just wanted to make sure you wanted it in.